docs/designers-developers/developers/tutorials/create-block/readme.md

```diff
@@ -20,13 +20,18 @@ You now need to activate the plugin from inside wp-admin plugins page.
 
 After activated, go to the block editor and use the inserter to search and add your new block.
 
+## Prerequisities
+
+The first thing you need is a development enviornment and tools. This includes setting up your WordPress environment, Node, NPM, and your code editor. If you need help, see the [setting up your development environment documentation](/docs/designers-developers/developers/tutorials/devenv/readme.md).
+
 ## Table of Contents
 
-1. [Development Environment](devenv.md)
-2. [WordPress Plugin](wp-plugin.md)
-3. [ESNext Syntax](esnext-js.md)
-4. [Anatomy of a Gutenberg Block](block-anatomy.md)
-5. [Block Attributes](block-attributes.md)
-6. [Code Implementation](block-code.md)
-7. [Authoring Experience](author-experience.md)
-8. [Finishing Touches](finishing.md)
+The create a block tutorials breaks down to the following sections.
+
+1. [WordPress Plugin](wp-plugin.md)
```

Comment:
Don't we need full URLs here to make sure they work at https://developer.wordpress.org/block-editor/?

```suggestion
1. [WordPress Plugin](/docs/designers-developers/developers/tutorials/create-block/wp-plugin.md)
```

and so on